test/unit/mocks/mock_http_test.dart

```diff
@@ -54,15 +54,6 @@ void main() {
             throwsA(isA<transport.RequestException>()));
       });
 
-      test('verifies that requests are mock requests before controlling them',
-          () {
-        transport.BaseRequest? request;
-        expect(() {
-          // ignore: deprecated_member_use_from_same_package
-          MockTransports.http.completeRequest(request);
-        }, throwsArgumentError);
-      });
```

Comment:
This was the test above verifying `completeRequest` can be passed a null. `completeRequest` isn't exported, and its only used in test files